CHANGELOG

```diff
@@ -1,3 +1,20 @@
+snowplow-ecommerce 0.7.0 (2024-01-19)
+---------------------------------------
+## Summary
+This release provides a solution for when multiple events are sent for a single transaction. 
+
+From this release onwards, the package is being released under the [Snowplow Community License, Version 1.0](https://docs.snowplow.io/community-license-1.0/). If you are uncertain how it applies to your use case, check our answers to [frequently asked questions](https://docs.snowplow.io/docs/contributing/community-license-faq/).
```

Comment:
This one is still community!

Reply:
Sorry, copied that from the web change log, thank you!